docs/_dapp/native.md

```diff
@@ -3,64 +3,89 @@ title: Go API
-it became apparent that providing programmatic access to the web3 pillars would open the
-door towards a new class of applications. As such, the second incarnation of the web3
-dream is to open up all our technologies for other projects as reusable components.
+Programmatic access to client functions is best delivered in the form of an open API. 
+This not only levels-up the complexity of dapps that can be built on top of 
+Ethereum but also opens up client technologies as re-usable, composable units that can be 
+applied in creative ways by a global community of developers.
 
-Starting with the 1.5 release family of `go-ethereum`, we transitioned away from providing
-only a full blown Ethereum client and started shipping official Go packages that could be
-embedded into third party desktop and server applications.
+To support this vision, Geth ships official Go packages that can be embedded into third party 
+desktop and server applications. There is also a [mobile API](/docs/dapp/mobile) that can be 
+used to embed Geth into mobile applications.
 
-*Note, this guide will assume you are familiar with Go development. It will make no
-attempts to cover general topics about Go project layouts, import paths or any other
-standard methodologies. If you are new to Go, consider reading its [getting started guides][go-guide] first.*
+This page provides a high-level overview of the Go API.
 
-## Quick overview
+*Note, this guide will assume some familiarity with Go development. It does not cover general topics 
+about Go project layouts, import paths or any other standard methodologies. If you are new to Go, 
+consider reading [Getting Started with Go][go-guide] first.*
 
-Our reusable Go libraries focus on four main usage areas:
+## Overview
+
+Geth's reusable Go libraries focus on three main usage areas:
 
 - Simplified client side account management
 - Remote node interfacing via different transports
 - Contract interactions through auto-generated bindings
-- In-process Ethereum, Whisper and Swarm peer-to-peer node
 
-You can watch a quick overview about these in Peter's (@karalabe) talk titled "Import
-Geth: Ethereum from Go and beyond", presented at the Ethereum Devcon2 developer conference
-in September, 2016 (Shanghai). Slides are [available here][peter-slides].
+The libraries are updated synchronously with the Geth Github repository. 
+The Go libraries can be viewed in full at [Go Packages][go-pkg-link]
+
+The benefits of using the Go libraries include easy client-side [account management](/docs/dapp/native-accounts)
+that allows users to keep their private keys safely encrypted locally and make their own
+security decisions. The Go libraries allow dapp developers to spend more of their time
+building their dapps rather than implementing complex protocols such as encoding and
+decoding function calls to smart contracts - this can all be abstracted away using
+Geth's Go libraries.
+
```

Comment:
This looks to me like a sales pitch. I don't see why we need to pitch people to use our libraries for encoding/decoding.
 
I'd prefer less text, and would rather remove blurbs like these which doesn't provide information.